opensrp-pnc/src/main/java/org/smartregister/pnc/repository/PncRegistrationDetailsRepository.java

```diff
@@ -67,12 +67,14 @@ public String getTableName() {
         try {
             if (StringUtils.isNotBlank(baseEntityId)) {
                 return rawQuery(getReadableDatabase(),
-                        "select * from " + PncDbConstants.Table.PNC_REGISTRATION_DETAILS +
-                                " where " + PncDbConstants.Column.PncDetails.BASE_ENTITY_ID + " = '" + baseEntityId + "' limit 1").get(0);
+                        "SELECT *, pvi.created_at AS latest_visit_date, prd.base_entity_id as base_entity_id, prd._id AS _id FROM " + PncDbConstants.KEY.TABLE + " AS ec \n" +
```

Comment:
create a method name that is more descriptive .... and add these changes

Reply:
method removed due to unused